packages/cms/pages/resources/resource/resource.tsx

```diff
@@ -241,13 +263,48 @@ export const Table: React.FunctionComponent = () => {
             icon: 'trash',
             type: 'icon',
             color: 'danger',
-            onClick: console.log
+
+            onClick: item => {
+              setIsModalVisible(true)
+              setDeleteId([item.id])
+            }
           }
         ]
       }
     ]
   }, [resource])
 
+  const ConfirmDeleteModal = () => {
+    const closeModal = () => setIsModalVisible(false)
+
+    if (isModalVisible) {
+      return (
+        <EuiConfirmModal
+          title={`Do you want to delete the selected item${
```

Comment:
This text should be customised to match the current resource. So `the selected products` or `the selected collections` etc.